Samples/LambdaLang/Language/Lit.cs

```diff
@@ -52,8 +52,15 @@ public IS<Lit, T> LitI(int value)
         => Lit.LitI<T>(value);
 }
 
-sealed class LitMapSemantic<TS, TR>(Func<TS, TR> F) : ILitSemantic<TS, IS<Lit, TR>>
+sealed class LitMapSemantic<TS, TR> : ILitSemantic<TS, IS<Lit, TR>>
 {
+    private readonly Func<TS, TR> f;
```

Comment:
Since f is not used, just remove it from constructor arguments, in this case LitMapSemantic will don't need constructor arguments. Don't save it in private property